daisy_workflows/image_build/enterprise_linux/kickstart/rhel_8_10_sap.cfg

```diff
@@ -30,7 +30,7 @@ udevadm control --reload
 udevadm trigger --settle
 tee -a /tmp/disk-config << EOM
 # build_installer.py will replace with the id of the install disk to avoid race conditions
-bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0 --append="net.ifnames=0 biosdevname=0 scsi_mod.use_blk_mq=Y"
+bootloader --boot-drive=/dev/disk/by-id/google-el-install-disk --timeout=0"
```

Comment:
extra `"`

Reply:
Done (pretty sure I did that last time too)